src/pages/MyReservations.js

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+
+const MyReservations = () => {
```

Comment:
- [ ] To go according to the new requirements, kindly change the `MyReservation` page to `MyFavorites` where a user sees all information about the favourited car. eg: (Car name, model, color, price, etc)